File: js-test/input/indent.c
// :code-block-start:SomeTest_Example
class OneOff : Object {
    // ...members...
}
// :hide-start:
// hide this cruft
class SomeTest : XCTestCase {
    func testOneOff() {
        // :hide-end:
        // later ...
        let oneOff = OneOff()
        // use oneOff ...
        // :code-block-end:
    }
}

File: js-test/expected/indent.js
module.exports.output = {
   start: [
    'class OneOff : Object {',
    '    // ...members...',
    '}',
    '        // later ...',
    '        let oneOff = OneOff()',
    '        // use oneOff ...',
    '    }',
    '}',
    ''
 ], final: [
   'class OneOff : Object {',
   '   // ...members...',
   '}',
   '// hide this cruft',
   'class SomeTest : XCTestCase {',
   '    func testOneOff() {',
   '        // later ...',
   '        let oneOff = OneOff()',
   '        // use oneOff ...',
   '    }',
   '}',
   ''
 ]
};